dashboard/app/helpers/pd/workshop_survey_results_helper.rb

```diff
@@ -219,13 +219,28 @@ def generate_workshops_survey_summary(workshop, questions)
               session_summary[response_section][q_key] = sum
             end
           else
+            # For bubble responses, return a frequency map with nulls removed
+            # [1, 1, 2, 2, 3, 5, 7, 7, 7, 7, 7, nil, nil] => {1: 2, 2: 2, 3: 1, 5: 1, 7: 5}
+            #
+            # For facilitator-specific responses, return a map per facilitator:
+            # { "Facilitator Name 1": {1: 2, 2: 2, ...}, "Facilitator Name 2": ... }
             if response_section == :facilitator
-              # Facilitator specific multiple choice answers are not currently supported
-              next
+              facilitator_responses = Hash.new
+              surveys_for_session[:facilitator]&.each do |survey|
+                next unless survey[q_key].presence
+                facilitator_responses[survey['facilitatorId'].to_i] = (facilitator_responses[survey['facilitatorId'].to_i] || []).append survey[q_key]
+              end
+
+              if current_user&.facilitator?
```

Comment:
Maybe this should be if the current user is not a program manager, workshop organizer, or workshop admin?

Also I think the ampersand is not needed here - users need to be logged in to access this page

Reply:
Now I'm wondering why I did that...but the change should be applied in both sections